packages/block/test/testdata/testdata-from-rpc.json

```diff
@@ -1,5 +1,4 @@
 {
-  "author": "0x4bb96091ee9d802ed039c4d1a5f6216f90f81b01",
```

Comment:
We also have those non-standard fields (e.g. author, creates, etc.) in other `from-rpc` testdata JSON files. I have no insight into why these are there, but from what I've read they are part of the Ethereum JSON-RPC standard (see https://ethereum.github.io/execution-apis/api-documentation/) 

![image](https://user-images.githubusercontent.com/18757482/188213457-86c23500-893e-41f9-be3f-3d6001051851.png)

Could we completely remove them? If so, we could as remove the `as unknown as JsonRpcBlock` typecastings from the `from-rpc` test file.

Reply:
I think we could.  Only the ones you have expanded there are in the standard responses I see when interacting with providers.  Should I go ahead and do it in this PR?

Reply:
Extracted some of your work + additional cases in the PR here: https://github.com/ethereumjs/ethereumjs-monorepo/pull/2281